emission/tests/analysisTests/TestClusterPipeline.py

```diff
@@ -1,171 +0,0 @@
-import unittest
-import emission.core.get_database as edb
-import emission.analysis.modelling.tour_model.cluster_pipeline as cp
-import emission.simulation.trip_gen as tg
-import uuid as uu
-
-class ClusterPipelineTests(unittest.TestCase):
```

Comment:
Okay!

On Thursday, April 21, 2016, shankari <notifications@github.com> wrote:

> In emission/tests/analysisTests/TestClusterPipeline.py
> <https://github.com/e-mission/e-mission-server/pull/263#discussion_r60697213>
> :
>
> > @@ -1,171 +0,0 @@
> > -import unittest
> > -import emission.core.get_database as edb
> > -import emission.analysis.modelling.tour_model.cluster_pipeline as cp
> > -import emission.simulation.trip_gen as tg
> > -import uuid as uu
> > -
> > -class ClusterPipelineTests(unittest.TestCase):
>
> Note that I have made substantial changes to the codebase in terms of
> storing trips and places. Please remember to pull and switch to a new
> branch before making any more changes. #265
> <https://github.com/e-mission/e-mission-server/pull/265> Shankari
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/e-mission/e-mission-server/pull/263/files/599f341f0bacc3ce6e5843b5c861a0ff7710a049#r60697213>
>